cpu/stm32/include/periph/cpu_dma.h

```diff
@@ -69,6 +69,26 @@ typedef enum {
     DMA_MEM_TO_MEM = 2,        /**< Memory to memory */
 } dma_mode_t;
 
+/**
+ * @brief   Burst Transfer modes for F2/F4/F7
+ */
+typedef enum {
+    DMA_BURST_SINGLE = 0,   /**< single transfer */
+    DMA_BURST_INCR4 = 1,    /**< incremental burst of 4 beats */
```

Comment:
Beats or bits ?

Reply:
Beats, from [Reference Manual for STM32F75xxx and STM32F74xxx](https://www.st.com/resource/en/reference_manual/rm0385-stm32f75xxx-and-stm32f74xxx-advanced-armbased-32bit-mcus-stmicroelectronics.pdf), Section 8.5.5, page 246:

Bits 24:23 **MBURST[1:0]**: memory burst transfer configuration
   These bits are set and cleared by software.
   00: single transfer
   01: INCR4 (incremental burst of 4 beats)
   10: INCR8 (incremental burst of 8 beats)
   11: INCR16 (incremental burst of 16 beats)
These bits are protected and can be written only if EN is ‘0’.
In direct mode, these bits are forced to 0x0 by hardware as soon as bit EN= '1'.
```